corehq/apps/users/models.py

```diff
@@ -230,6 +230,11 @@ class UserRole(Document):
     permissions = SchemaProperty(Permissions)
     is_archived = BooleanProperty(default=False)
 
+    @classmethod
+    @quickcache(['doc_id'], timeout=0)
+    def get(cls, doc_id):
```

Comment:
only comment is that it would be nice if we could replace the half of `CachedCouchDocumentMixin` that is shared with the other one but not sure how achievable that will be. just always nice to have a single recommended way to do things.